src/components/helperFunctions.tsx

```diff
@@ -0,0 +1,23 @@
+export function countTextLines(text: string) {
+    const canvas = document.createElement("canvas");
+    const context = canvas.getContext("2d") as CanvasRenderingContext2D;
```

Comment:
**thought:** I guess it's GPT idea :D But using canvas for that problem? I think it's overcomplicated, does not have textarea some attributes for that?